### DISCUSSION TITLE
ZuluSCSI RP2040 compact and an Atari Mega STE

### DISCUSSION THREAD

**Discussion by vad4r** [ZuluSCSI/ZuluSCSI-firmware#369] [Q&A]
Hello,

I would like to use my ZuluSCSI RP2040 compact on an Atari Mega STE.
This can only manage 1 GB hard disks and these need SCSI ID0 to be recognized. So I have created HD0.img, with sizes from 90MB to 1GB. 
I tried AHDI and ICD as drivers. Both find something on ID0 when booting, but they can't do anything with it:
AHDI (ID0: identification unaviable)
ICD (ID0: unreconized unit)

I'm using a 2GB SD card, but that shouldn't matter?

So it seems that a few parameters are still incorrect. I am simply overwhelmed with a configured zuluscsi.ini, too much information with which I can do nothing. Therefore I hope that a successful user can provide me with this information?

What about parity, is it always on or how can I configure it, it has to be off.
The termination must also be on, but the Atari does not provide a signal (therefore it must be active (?))
Would be great if you could help me, I bought the Zulu especially for this.

**Comment by morio** [comment]
@vad4r 
There is a flag that should work.
```
[SCSI]
System = "MegaSTE"
```

The 2GB card is probably pretty old and there have been some issues with older cards on some of the ZuluSCSIs. 
If you cold post the `zululog.txt` it might shed some light on what is going on, after turning the `DEBUG LOG` dip switch to `ON`. If there isn't a `zululog.txt` it probably means your SD card isn't compatible and consider getting a recent SanDisk.

**Reply by vad4r** [reply]
Thank you very much for your comment. You didn't give me the solution directly, but you inspired me to try it another way - and it works, I'm very happy.

I am now wondering if there is a way to exchange data between PC and Atari using the SD card. I have an image file on it now, but I can't edit it on the PC, can I?

**Reply by aperezbios** [reply]
You can, assuming you're using file-based images, but you'd need software that's capable of handling Atari-style FAT partitions/filesystems, under whatever OS you're using. I'd be surprised if something like that didn't exist, but it certainly falls outside of the scope of ZuluSCSI support. If you find something, feel free to share it here.

**Reply by Trollicus-Rex** [reply]
> Thank you very much for your comment. You didn't give me the solution directly, but you inspired me to try it another way - and it works, I'm very happy.
> 
> I am now wondering if there is a way to exchange data between PC and Atari using the SD card. I have an image file on it now, but I can't edit it on the PC, can I?
Do you mind sharing how you got it working??

**Reply by dmsdms2000** [reply]
Vad4r says tried something else but got it working, would be very interested in what is needed to get zuluscsi working in a Supra enclosure on an Atari st.

**Reply by fdanapfel** [reply]
> I am now wondering if there is a way to exchange data between PC and Atari using the SD card. I have an image file on it now, but I can't edit it on the PC, can I?

You could use an Atari emulator like [Hatari](https://www.hatari-emu.org) on your PC to access the image file on the SD card by attaching it as an ACSI disk on an emulated MegaSTE.

The GEMDOS drive mode of Hatari then allows you to access a folder on your PC from inside the emulated MegaSTE to copy files to and from the image file.
